packages/clerk-js/src/ui/common/constants.ts

```diff
@@ -96,6 +96,19 @@ export function getWeb3ProviderData(name: Web3Provider): Web3ProviderData | unde
   return WEB3_PROVIDERS[name];
 }
 
+/**
+ * Returns the URL for a static SVG image
+ * using the old images.clerk.com service
+ * @deprecated In favor of iconImageUrl
+ */
 export function svgUrl(id: string): string {
   return `https://images.clerk.com/static/${id}.svg`;
 }
+
+/**
+ * Returns the URL for a static SVG image
+ * using the new img.clerk.com service
+ */
+export function iconImageUrl(id: string): string {
+  return `https://img.clerk.com/static/${id}.svg`;
```

Comment:
I know it was like that before as well, but shouldn't this be different for staging?

Reply:
This can be done when we start splitting the bundles based on the build target, right now I'm not sure if it's worth propagating this info to this specific function as the logic and the URL won't really change.

Reply:
@nikosdouvlis That's because static assets/files will eventually be the same on both staging and production environments, right?

Reply:
Yes. Actually, we only upload them to the prod bucket right now (it's just the oauth provider logos)